frontend/javascripts/dashboard/dataset/dataset_import_view.js

```diff
@@ -65,8 +70,8 @@ type State = {
   activeTabKey: TabKey,
   savedDataSourceOnServer: ?APIDataSource,
   inferredDataSource: ?APIDataSource,
-  differenceBetweenDatasources: ?Object,
-  didApplyDifferenceBetweenDatasources: boolean,
+  differenceBetweenDataSources: ?Object,
```

Comment:
```suggestion
  differenceBetweenDataSources: Object,
```

At least from what I can see.